Telegram/SourceFiles/config.h.in

```diff
@@ -251,7 +251,7 @@ static const char *ApiHash = "344583e45741c457fe1862106095a5eb";
 #endif
 
 #if Q_BYTE_ORDER == Q_BIG_ENDIAN
-#error "Only little endian is supported!"
+#warning "Only little endian is supported!"
```

Comment:
Wait, does it mean we support big endian now? Maybe delete the warning then?

Reply:
We need to test it first.

Reply:
Alright, then I suggest to change the text: "WARNING: Big endian support in Kepka is experimental"

It should be clear from the warning that
- we're working on it and we aren't sure if it works or not
- what application causes this warning (could be useful for cases when our code is transitive dependency of something else)